table/block_based_table_reader.cc

```diff
@@ -2062,6 +2062,7 @@ Status BlockBasedTable::Get(const ReadOptions& read_options, const Slice& key,
   if (!FullFilterKeyMayMatch(read_options, filter, key, no_io)) {
     RecordTick(rep_->ioptions.statistics, BLOOM_FILTER_USEFUL);
   } else {
+    RecordTick(rep_->ioptions.statistics, BLOOM_FILTER_FULL_POSITIVE);
```

Comment:
Shall we log both of BLOOM_FILTER_FULL_POSITIVE and BLOOM_FILTER_FULL_TRUE_POSITIVE only when filter exists, i.e. filter != nullptr?

Reply:
The BLOOM_FILTER_USEFUL counter was useful to me at one time to figure out the bloom filter is not enabled (when it =0). I think we should make the filter counters consistently being 0 if filter non-exists.

Reply:
but it will look confusing for a normal user if he don't set bloom filter but still see non-zero filter counters..